Source/Core/UICommon/NetPlayIndex.cpp

```diff
@@ -318,3 +318,8 @@ const std::string& NetPlayIndex::GetLastError() const
 {
   return m_last_error;
 }
+
+bool NetPlayIndex::HasActiveSession() const
+{
+  return !m_secret.empty();
```

Comment:
This doesn't properly indicate that the session is still active, as the notification loop can exit due to failure and `m_secret` will still be non-empty. Should probably make it empty when the notification loop returns.